docs/installation/observability/metrics.rst

```diff
@@ -0,0 +1,237 @@
+    enabled, but the code is in the Open Telemetry SDK instrumentation already and could
+    possibly be opted-in to.
+
+Application specific
+====================
+
+Accounts
+--------
+
+``user_count``
+    Reports the number of users in the database. This is a global metric, you must take
+    care in de-duplicating results. Additional attributes are:
+
+    - ``scope`` - fixed, set to ``global`` to enable de-duplication.
+    - ``type`` - the user type. ``all``, ``staff`` or ``superuser``.
+
+    Sample PromQL query:
+
+    .. code-block:: promql
+
+        max by (type) (last_over_time(
+          otel_user_count{scope="global"}
+          [1m]
+        ))
+
+``auth.login_failures``
+    A counter incremented every time a user login fails (typically because of invalid
+    credentials). Does not include the second factor, if enabled. Additional attributes:
+
+    - ``http_target`` - the request path where the login failure occurred, if this
+      happened in a request context.
+
+``auth.user_lockouts``
+    A counter incremented every time a user is locked out because they reached the
+    maximum number of failed attempts. Additional attributes:
+
+    - ``http_target`` - the request path where the login failure occurred, if this
+      happened in a request context.
+    - ``username`` - username of the user trying to log in.
+
+``auth.logins``
+    Counter incrementing on every successful login by a user. Additional attributes:
+
+    - ``http_target`` - the request path where the login failure occurred, if this
+      happened in a request context.
+    - ``username`` - username of the user trying to log in.
+
+``auth.logouts``
+    Counter incrementing every time a user logs out. Additional attributes:
+
+    - ``username`` - username of the user who logged out.
+
+Klantcontacten
+--------------
+
+``klantcontacten.create``
+    Reports the number of klantcontacten created via the API.
+    Additional attributes are:
+
+    - ``uuid`` – unique identifier of the Klantcontact.
+    - ``nummer`` – the reference number of the Klantcontact.
+
+``klantcontacten.update``
+    Reports the number of klantcontacten updated via the API.
+    Additional attributes are:
+
+    - ``uuid`` – unique identifier of the Klantcontact.
+    - ``nummer`` – the reference number of the Klantcontact.
+
+``klantcontacten.delete``
+    Reports the number of klantcontacten deleted via the API.
+    Additional attributes are:
+
+    - ``uuid`` – unique identifier of the Klantcontact.
+    - ``nummer`` – the reference number of the Klantcontact.
+
+Betrokkenen
+-----------
+
+``betrokkenen.create``
+    Reports the number of betrokkenen created via the API.
+    Additional attributes are:
+
+    - ``uuid`` – unique identifier of the Betrokkene.
+    - ``partij_uuid`` – unique identifier of the related Partij.
+    - ``klantcontact_uuid`` – unique identifier of the related Klantcontact.
```

Comment:
Same concern here about the cardinality

In Open Forms, we don't expect more than 10-100 forms to exist, but a `partij` can in theory point to every citizen in NL which gives you a potential cardinality of 17 million - that's extremely high :)